### ISSUE TITLE
Message stream is not used when specified

### ISSUE THREAD

**Issue by anark** [wildbit/postmark-rails#81]
I tried sending with the following format but sends are always using the default transactional message stream when it should use the stream specified.

```
  def my_email(name)
    metadata['name'] = name
    mail(to: 'bob@example.com', subject: 'Terms Updated', message_stream: 'my-broadcast-stream')
  end
```

**Comment by anark** [comment]
I now see this is already possible with the following
```
  def my_email(name, message_stream)
    metadata['name'] = name
    mail(to: 'bob@example.com', subject: 'Terms Updated', message_stream: message_stream)
  end
```

**Comment by anark** [comment]
I tried sending with the following format but sends are always using the default transactional message stream

```
  def my_email(name)
    metadata['name'] = name
    mail(to: 'bob@example.com', subject: 'Terms Updated', message_stream: 'my-broadcast-stream')
  end
```

**Comment by anark** [comment]
With debugging I have found that the message pasted to postmark gem at line https://github.com/wildbit/postmark-gem/blob/master/lib/postmark/mail_message_converter.rb#L38 is missing the @message.message_stream but does contain @message['message-stream'].  Any ideas?

**Comment by nickhammond** [comment]
Hey @anark! Thanks for bringing this to our attention, this has been fixed in [1.21.1](https://rubygems.org/gems/postmark/versions/1.21.2). Can you give it a whirl and re-open this if you're still having issues? Going to close this for now.
